modules/virt-checking-storage-configuration.adoc

```diff
@@ -166,15 +167,17 @@ data:
   status.failureReason: "" # <2>
   status.startTimestamp: "2023-07-31T13:14:38Z" # <3>
   status.completionTimestamp: "2023-07-31T13:19:41Z" # <4>
-  status.result.cnvVersion: 4.16.2
-  status.result.defaultStorageClass: trident-nfs <5>
-  status.result.goldenImagesNoDataSource: <data_import_cron_list> # <6>
-  status.result.goldenImagesNotUpToDate: <data_import_cron_list> # <7>
-  status.result.ocpVersion: 4.16.0
-  status.result.storageMissingVolumeSnapshotClass: <storage_class_list> 
-  status.result.storageProfilesWithEmptyClaimPropertySets: <storage_profile_list> # <8>
-  status.result.storageProfilesWithSpecClaimPropertySets: <storage_profile_list> 
-  status.result.storageWithRWX: |-
+  status.result.cnvVersion: 4.16.2 # <5>
+  status.result.defaultStorageClass: trident-nfs <6>
+  status.result.goldenImagesNoDataSource: <data_import_cron_list> # <7>
+  status.result.goldenImagesNotUpToDate: <data_import_cron_list> # <8>
+  status.result.ocpVersion: 4.16.0 # <9>
+  status.result.pvcBound: "true" # <10>
+  status.result.storageProfileMissingVolumeSnapshotClass: <storage_class_list> # <11>
```

Comment:
<storage_profile_list>

Reply:
Right, but storage classes and their profiles has the same name so not sure how important is it.

Reply:
I'll change this to <storage_profile_list>